packages/react-theme-provider/src/__snapshots__/ThemeProvider.test.tsx.snap

```diff
@@ -60,11 +55,6 @@ exports[`ThemeProvider can apply body theme to body 2`] = `
   className=
       foo
       {
-        --body-fontFamily: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
-        --body-fontSize: 14px;
-        --body-fontWeight: 400;
-        --body-mozOsxFontSmoothing: grayscale;
-        --body-webkitFontSmoothing: antialiased;
```

Comment:
why aren't these defined?

Reply:
we can keep these, but they are not being used anywhere